frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -446,12 +450,33 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
                 </Card>
               </Radio.Button>
             </Tooltip>
+            <Tooltip title="Coming soon">
+              <Radio.Button
+                className="aIJobSelection"
+                checked={aIJobModalState === "align_sections"}
+                disabled={!Store.getState().activeUser?.isSuperUser}
+                onClick={() => Store.dispatch(setAIJobModalStateAction("align_sections"))}
```

Comment:
And here is still a`Store.dispatch`
```suggestion
                onClick={() => dispatch(setAIJobModalStateAction("align_sections"))}
```